base/ltfssaxes.dtx

```diff
@@ -154,30 +155,85 @@
 %
 %    We make \texttt{m} to reset both weight and width (as this is
 %    how it always worked). To reset just the width \texttt{?m} is
-%    provided and to reset just the weight \texttt{m?}.
+%    provided and to reset just the weight \texttt{m?}\ is provided.
 %
 % \item
 %
-%   We do support ``\texttt{m}\textit{width}'' and
-%    ``\textit{weight}\texttt{m}'', e.g., \texttt{mec} to mean ``go to
+%    We do support ``\texttt{m}\meta{width}'' and
+%    ``\meta{weight}\texttt{m}'', e.g., \texttt{mec} to mean ``go to
 %    medium weight and extra-condensed width''. At the end of the
 %    process we automatically drop any leftover \texttt{m} in the
 %    series name (unless it is just a single \texttt{m}).
 %
+% \item
+%
+%    If there is no table entry then the requested series is used
+%    unconditionally. This means that we usually do not need entries
+%    where the second argument (the requested series) and third argument
+%    (the result series) are identical (unless we want to use the fourth
+%    argument to specify an alternative result series). In particular,
+%    this means:
+%    \begin{itemize}
+%    \item
+%
+%      Any request for \texttt{m} needs no entry, i.e., there are no entries
+%      which have \texttt{m} as second argument.
+%
+%    \item
+%
+%       Any request to set both weight and width (e.g., \texttt{sbx} or
+%       \texttt{ulc}) needs no entry. For that reason, there are no entries
+%       which have a weight+width as second argument (except for a few
+%       cases where the fourth argument is used to specify an alternative
+%       result series). In particular, this is also true for cases involving
+%       \texttt{m}, e.g., \texttt{bm} (bold medium width) which
+%       automatically gets reduced to \texttt{b}, or \texttt{mc} (medium
+%       weight condensed) which becomes \texttt{c} as a result.
```

Comment:
```suggestion
%       which have a weight+width as second argument (except for cases
%       involving \texttt{bx}, see below). In particular, this is also true
%       for cases involving \texttt{m}, e.g., \texttt{bm} (bold medium width)
%       which automatically gets reduced to \texttt{b}, or \texttt{mc}
%      (medium weight condensed) which becomes \texttt{c} as a result.
```

Reply:
don't think that's  better to be honest. It isn't just bx.

Reply:
Unless I'm missing something, there are actually no other examples of "the second argument contains weight+width" than `bx`...